lib/charms/sdcore_nms_k8s/v0/fiveg_core_gnb.py

```diff
@@ -0,0 +1,416 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the `fiveg_core_gnb` relation.
+
+This library contains the Requires and Provides classes for handling the `fiveg_core_gnb`
+interface.
+
+The purpose of this library is to provide a way for a 5G core to provide network information and
```

Comment:
```suggestion
The purpose of this library is to provide a way for a 5G Core to provide network information and
```
5G Core is the official name of the core in 5G ;)